lib/capistrano/tasks/deploy.rake

```diff
@@ -3,6 +3,27 @@ namespace :deploy do
     invoke "sidekiq:add_default_hooks"
   end
 
+  desc "Fix Bundler plugin path so it points to the shared path instead of a release path"
+  task :fix_bundler_plugin_path do
+    on release_roles([:all]) do
+      within shared_path do
+        # sed -i 's#/home/deploy/apps/simple-server/releases/[0-9]\+/.bundle/#/home/deploy/apps/simple-server/shared/.bundle/#g' plugin/index
+        execute "sed", "-i", '"s#/home/deploy/apps/simple-server/releases/[0-9]\+/#/home/deploy/apps/simple-server/shared/#g"', ".bundle/plugin/index"
```

Comment:
We need a carefully crafted sed command here to replace the release dir with the shared dir, but this finally works.